astropy/coordinates/tests/test_name_resolve.py

```diff
@@ -114,9 +114,9 @@ def test_names():
     # "all" choice should ask for SIMBAD, then NED, then Vizier: "SNV" in the url
     with pytest.raises(
         NameResolveError,
+        # avoid hard-coding the exact url as it might depend on external state
         match=re.escape(
-            "Unable to find coordinates for name 'm87h34hhh' "
-            "using https://cds.unistra.fr/cgi-bin/nph-sesame/SNV?m87h34hhh"
+            "Unable to find coordinates for name 'm87h34hhh' " "using http"
```

Comment:
```suggestion
            "Unable to find coordinates for name 'm87h34hhh' using http"
```
Furthermore, there's no need to call `re.escape()` anymore.

Reply:
nice catch. Done !